### ISSUE TITLE
BLorient586

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#2157]
I'm having trouble with the description of the content of BLorietn586 (current state in branch DR_22_05_BLorient586):
https://archive.org/details/catalogueofethio00brituoft/page/116/mode/2up?view=theater
http://www.bl.uk/manuscripts/Viewer.aspx?ref=or_586_f001r
The first two of Wright's items are probably LIT3186Meeraf and LIT1990Mawase, but I'm not sure whether the third is LIT2610Zemmar, and also don't know how to deal with the text on f. 111. 
Any suggestions?
@DenisNosnitsin1970 @eu-genia

**Comment by eu-genia** [comment]
Hmm. I am not sure that 111rv are the same hand. Since there are several blank leaves following the second text it may well be that the indications are a later addition and not an msItem.
It does not look like LIT2610Zemmar either. Could Jonas know the texts?

**Comment by eu-genia** [comment]
(the note on 128v is also a later addition AFAIK)

**Comment by DenisNosnitsin1970** [comment]
Yes, it seems to be Weddase Maryam with musical notes, we probably have an ID for that
.

**Comment by DenisNosnitsin1970** [comment]
It goes up to fol. 118vb, however, followed by other hymns, for the rest you can check መስተግብዕ, EMIP02036 is close in content to Orient 586

**Comment by thea-m** [comment]
Thank you, yes, 113-128 are LIT6489SchoolChants! 129ra-195vc are still unclear to me, I will note a reference with uncertainty to LIT2610Zemmar. I tried to check Velat's edition, which lists this manuscript among those used, but found no reference to these folia.

**Comment by DenisNosnitsin1970** [comment]
129ra-195vc is Zemmare, you can check its beginning against a regular Zemmare ms or a church edition, it matches. Possibly, again, it is a special version of it.

**Comment by eu-genia** [comment]
so the addition on 128v refers to the following leaves, not the preceding ones
